qemu/tests/virtual_nic_private.py

```diff
@@ -68,6 +85,20 @@ def data_mon(session, cmd, timeout):
         else:
             if_func = utils_net.get_windows_nic_attribute
             args = (mon_session, "macaddress", mon_macaddr, "netconnectionid")
+            error_context.context("Install wireshark", logging.info)
+            install_wireshark_cmd = params.get("install_wireshark_cmd")
+            install_wireshark_cmd = utils_misc.set_winutils_letter(
+                    sessions[2], install_wireshark_cmd)
+            status, output = sessions[2].cmd_status_output(install_wireshark_cmd,
+                                                           timeout=timeout)
+            if status:
+                test.error("Failed to install wireshark, status=%s, output=%s"
+                           % (status, output))
+            logging.info("Wait for wireshark installation to complete")
+            utils_misc.wait_for(
```

Comment:
Is `c:\${wireshark_name} /S` running in the background so need to continue to check the installer at the next step?

Reply:
Hi @PaulYuuu  I think this is necessary. Check “{wireshark_name}” is actually to give some installation time.

Reply:
Could you get the output of `c:\${wireshark_name} /S`, if it runs in the background, then the next step is necessary.

Reply:
Hi @PaulYuuu I confirmed that “c:\${wireshark_name} /S“ is running in the background. And the output is empty in the test. So I think the next step is necessary.

Reply:
OK, thanks for the explanation.